modules/web/js/ballerina/ast/statements/statement.js

```diff
@@ -49,6 +49,60 @@ class Statement extends ASTNode {
     getStatementString() {
         throw new Error('This method should be implemented by sub class');
     }
+
+
+    /**
```

Comment:
@NatashaWso2 
You could include parameter type as @param {string} connectorName <description>
and returns type as @returns {object} <description>